src/views/certifications/views/[slug]/components/program-hero/index.tsx

```diff
@@ -5,18 +5,14 @@
 
 import classNames from 'classnames'
 // Shared under certifications
-import {
-	CertificationsHero,
-	CertificationsHeroText,
-	CertificationsHeroProps,
-} from 'views/certifications/components'
 import { ProgramSlug } from 'views/certifications/types'
 // Local
-import { ProgramHeroProps } from './types'
+import { ProgramHeroProps, HeroProps } from './types'
 // Styles
 import s from './program-hero.module.css'
 import StandaloneLink from 'components/standalone-link'
 import { IconArrowLeft16 } from '@hashicorp/flight-icons/svg-react/arrow-left-16'
+import { Hero, HeroText } from 'components/landing-hero/components/hero'
```

Comment:
Replaced local imports with the new location for these components in the global components dir